content/participants/ttjc@Rickey07/index.md

```diff
@@ -44,6 +44,14 @@ _Stack_ Made in Html , CSS and Vanilla Js.
 
 _Github link_ (https://github.com/Rickey07/MyallProjects/blob/master/Changehex.html)
 
+#### React_weather_app
+
+_description_ My First and simple web app for checking weather with dynamic background changing 
```

Comment:
Can you describe it in a better way like:
A react based web application that fetches live API data and shows weather conditions dynamically according to weather.